api/src/main/resources/liquibase.xml

```diff
@@ -460,4 +460,328 @@
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM privilege WHERE privilege='Manage Queue Rooms';
+            </sqlCheck>
+        </preConditions>
+        <comment>Add "Manage Queue Rooms" privilege</comment>
+        <insert tableName="privilege">
+            <column name="privilege" value="Manage Queue Rooms" />
+            <column name="description" value="Able to add/edit/retire queue rooms" />
+            <column name="uuid" value="a72dac37-b593-46a1-a24b-e3da6dbcc521" />
+        </insert>
+    </changeSet>
+
+    <changeSet id="rest_endpoints_accessed_without_authentication_2024050707" author="mujuzi" >
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM privilege WHERE privilege='Purge Queues';
+            </sqlCheck>
+        </preConditions>
+        <comment>Add "Purge Queues" privilege</comment>
+        <insert tableName="privilege">
+            <column name="privilege" value="Purge Queues" />
+            <column name="description" value="Able to delete queues from the database" />
+            <column name="uuid" value="1cb903f2-90df-11e0-9c16-705abgfds24r" />
+        </insert>
+    </changeSet>
+
+    <changeSet id="rest_endpoints_accessed_without_authentication_2024050708" author="mujuzi" >
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM privilege WHERE privilege='Purge Queue Entries';
+            </sqlCheck>
+        </preConditions>
+        <comment>Add "Purge Queue Entries" privilege</comment>
+        <insert tableName="privilege">
+            <column name="privilege" value="Purge Queue Entries" />
+            <column name="description" value="Able to delete queue entries from the database" />
+            <column name="uuid" value="22a5ea24-f755-42c2-ae56-b0d23gt567r4" />
+        </insert>
+    </changeSet>
+
+    <changeSet id="rest_endpoints_accessed_without_authentication_2024050709" author="mujuzi" >
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM privilege WHERE privilege='Purge Queue Rooms';
+            </sqlCheck>
+        </preConditions>
+        <comment>Add "Purge Queue Rooms" privilege</comment>
+        <insert tableName="privilege">
+            <column name="privilege" value="Purge Queue Rooms" />
+            <column name="description" value="Able to delete queue rooms from the database" />
+            <column name="uuid" value="9h8d710b-d333-40b7-b449-6e0e739d15d0" />
+        </insert>
+    </changeSet>
+
+    <changeSet id="rest_endpoints_accessed_without_authentication_20240507010" author="mujuzi" >
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <sqlCheck expectedResult="0">
+                    SELECT count(*) FROM role_privilege WHERE privilege='Get Visits';
```

Comment:
I would think the more appropriate check here, would be to execute this only if no one already has the "Get Queue Entries" privilege (which would imply it has already previously been assigned as desired).  Then, this also allows you to get rid of the "AND NOT EXISTS..." clause from your insert statement below, right?

The same sort of logic would be applied to all of the changesets below.  So:

* If any role already has privilege A, then mark as RAN
* Otherwise, assign privilege A to any role that already has privilege B.